src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -36,6 +36,10 @@ public void ApplyCapabilities(VSInternalServerCapabilities serverCapabilities, V
         {
             ResolveProvider = true,
             TriggerCharacters = _completionListProvider.AggregateTriggerCharacters.ToArray(),
+            // This is the intersection of C# and HTML commit characters.
+            // We need to specify it so that platform can correctly calculate ApplicableToSpan
```

Comment:
Might be more informative to link to the appropriate code for where `ApplicableToSpan` is important in the editor, as it doesn't mean anything to me, and our issue linked below doesn't shed any light on it.

Reply:
Added a link where we compute it using the commi character set (among other things). Do we also want a link where it gets converted to a text edit later (that was deleting the ">" mentioned in the bug? Seems like a lot of links :smile: but can add it needed

Reply:
Putting here anyway - the method that actually turns applicableToSpan into a text edit is https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform?path=/src/Editor/Language/Impl/Language/AsyncCompletion/AsyncCompletionSession.cs&version=GBmain&line=1207&lineEnd=1207&lineStartColumn=29&lineEndColumn=45&lineStyle=plain&_a=contents